runtime/onert/core/src/util/shapeinf/Pow.cc

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "util/ShapeInference.h"
+
+namespace onert
+{
+namespace shape_inference
+{
+
+void StaticInferer::visit(const ir::operation::Pow &op)
+{
+  const auto lhs_idx{op.getInputs().at(ir::operation::Pow::Input::LHS)};
+  const auto &lhs = _operands.at(lhs_idx);
+  const auto rhs_idx{op.getInputs().at(ir::operation::Pow::Input::RHS)};
+  const auto &rhs = _operands.at(rhs_idx);
+  const auto output_idx = op.getOutputs().at(0);
+  ir::Operand &output = _operands.at(output_idx);
+
+  if (lhs.info().isDynamic() || rhs.info().isDynamic())
+  {
+    output.info().setDynamic();
+    return;
+  }
+
+  // re-sizing output shape
+  ir::Shape new_shape = inferEltwiseShape(lhs.info().shape(), rhs.info().shape());
```

Comment:
`inferEltwiseShape` calculates output shape considering broadcasting.
I wonder if `pow` support broadcasting.

Reply:
I saw "pow_broadcast_float_nnfw.mod.py" source file.
It looks like broadcast test.
but I'm not sure about this ..
am I wrong ??

Reply:
It seems that you're correct. :-)

Reply:
thanks : )